src/main.ts

```diff
@@ -224,6 +226,7 @@ async function handleCanvasClick(event: MouseEvent): Promise<void> {
   if (id < 0) {
     selectedTrack = null;
     plot.removePlot();
+    selectedTrack = null; // clear selected track when clicking off of cells
```

Comment:
doing it twice?  or can you remove the line before removePlot?

Reply:
Oops, good catch!